gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinClusterConfigurationKeys.java

```diff
@@ -183,4 +185,11 @@
   public static final boolean DEFAULT_IS_HELIX_CLUSTER_MANAGED = false;
 
   public static final String HADOOP_CONFIG_OVERRIDES_PREFIX = GOBBLIN_CLUSTER_PREFIX + "hadoop.inject";
+
+  //Configuration keys that will be set dynamically.
+  public static final String HELIX_JOB_ID_KEY = GOBBLIN_CLUSTER_PREFIX + "helixJobId";
```

Comment:
Let's add different prefix to differentiate from exposed configuration ?

Reply:
Changed.